presto-main/src/test/java/com/facebook/presto/failureDetector/TestHeartbeatFailureDetector.java

```diff
@@ -88,6 +93,34 @@ public void configure(Binder binder)
         assertTrue(detector.getFailed().isEmpty());
     }
 
+    @Test
```

Comment:
@nezihyigitbasi thanks for the detailed feedback. I incorporated your suggestions to the commit I just made.